OpenOversight/app/main/views.py

```diff
@@ -481,12 +476,15 @@ def list_officer(department_id, page=1, race=[], gender=[], rank=[], min_age='16
     if request.args.get('rank') and all(rank in rank_choices for rank in request.args.getlist('rank')):
         form_data['rank'] = request.args.getlist('rank')
 
+    if request.args.get('year') and request.args.get('year') in year_choices():
```

Comment:
It looks like this was already happening the code base before you added this line, but I've always learned to use `is not None` to check for `None` in Python, as opposed to relying on the falsy-ness of `None`. This is recommended by PEP 8 (see the 2nd bullet point under the heading): https://www.python.org/dev/peps/pep-0008/#programming-recommendations

Reply:
TIL!